tests/test_build.py

```diff
@@ -281,6 +281,28 @@ def test_sidebars_level2(sphinx_build_factory, file_regression):
     file_regression.check(sidebar.prettify(), extension=".html")
 
 
+def test_sidebars_show_nav_level0_index(sphinx_build_factory, file_regression):
+    """"""
+    confoverrides = {"html_theme_options.show_nav_level": 0}
+    sphinx_build = sphinx_build_factory("sidebars", confoverrides=confoverrides).build()
+
+    # Both the column alignment and the margin should be changed
```

Comment:
I'm confused, why would the navigation depth effect column alignment and margin?